src/ray/raylet/node_manager.cc

```diff
@@ -326,6 +326,28 @@ void NodeManager::Heartbeat() {
     heartbeat_data->add_resource_load_capacity(resource_pair.second);
   }
 
+  std::unordered_set<ObjectID> active_object_ids = worker_pool_.GetActiveObjectIDs();
+  size_t max_size =
+      static_cast<size_t>(RayConfig::instance().raylet_active_object_ids_size());
+  if (active_object_ids.size() <= max_size) {
+    for (const auto &object_id : active_object_ids) {
+      std::string object_id_bytes = object_id.Binary();
+      heartbeat_data->add_active_object_id(object_id_bytes.data(),
+                                           object_id_bytes.size());
```

Comment:
I think you can do `to_flatbuf` here.

Reply:
This isn't flatbuf, it's protobuf. Same for other comment.